mod-mosaic/schemas/mosaic_order.json

```diff
@@ -24,9 +29,10 @@
       "description": "edition of the material",
       "type": "string"
     },
-    "productId": {
-      "description": "The actual product identifier",
-      "type": "string"
+    "details": {
+      "description": "details about this purchase order line",
+      "type": "object",
+      "$ref": "../../mod-orders-storage/schemas/details.json"
```

Comment:
We have receivingNote as separate field, and it is declared also in details, can we declare it in single place?

Reply:
also we forgot to delete folioOrderTemplateId previosly from this schema